README.rst

```diff
@@ -37,32 +37,56 @@ DALI in action::
   import nvidia.dali.types as types
   import nvidia.dali.fn as fn
   from nvidia.dali.plugin.pytorch import DALIGenericIterator
-
-  @pipeline_def(batch_size=128, num_threads=4, device_id=0)
-  def get_dali_pipeline(data_dir, crop_size):
-    images, labels = fn.readers.file(file_root=data_dir, shuffle=True, name="Reader")
-    # decode data on the GPU
-    images = fn.decoders.image_random_crop(images, device="mixed", output_type=types.RGB)
-    # the rest of processing happens on the GPU as well
-    images = fn.resize(images, resize_x=crop_size, resize_y=crop_size)
-    images = fn.crop_mirror_normalize(images,
-                                      mean=[0.485 * 255,0.456 * 255,0.406 * 255],
-                                      std=[0.229 * 255,0.224 * 255,0.225 * 255],
-                                      mirror=fn.random.coin_flip())
-    return images, label
-
+  import os
+  
+  # To run with different data, see documentation of nvidia.dali.fn.readers.file
+  # points to https://github.com/NVIDIA/DALI_extra
+  data_root_dir = os.environ['DALI_EXTRA_PATH']
+  images_dir = os.path.join(data_root_dir, 'db', 'single', 'jpeg')
+  
+  
+  def loss_func(pred, y):
+      pass
+  
+  
+  def model(x):
+      pass
+  
+  
+  def backward(loss, model):
+      pass
+  
+  
+  @pipeline_def(num_threads=4, device_id=0)
+  def get_dali_pipeline():
+      images, labels = fn.readers.file(
+          file_root=images_dir, random_shuffle=True, name="Reader")
+      # decode data on the GPU
+      images = fn.decoders.image_random_crop(
+          images, device="mixed", output_type=types.RGB)
+      # the rest of processing happens on the GPU as well
+      images = fn.resize(images, resize_x=256, resize_y=256)
+      images = fn.crop_mirror_normalize(
+          images,
+          crop_h=224,
+          crop_w=224,
+          mean=[0.485 * 255, 0.456 * 255, 0.406 * 255],
+          std=[0.229 * 255, 0.224 * 255, 0.225 * 255],
+          mirror=fn.random.coin_flip())
+      return images, labels
+  
+  
   train_data = DALIGenericIterator(
-     [get_dali_pipeline(data_dir, (244,244))], ['data', 'label’],
-     reader_name='Reader’
+      [get_dali_pipeline(batch_size=16)], ['data', 'label'],
```

Comment:
Readability - I thought that the entire line was one array, when in fact there are two.
```suggestion
      [get_dali_pipeline(batch_size=16)],
      ['data', 'label'],
```